tendermint/src/lite_impl/signed_header.rs

```diff
@@ -82,3 +72,19 @@ impl block::signed_header::SignedHeader {
             .collect()
     }
 }
+
+impl From<block::signed_header::SignedHeader>
+    for lite::types::SignedHeader<block::signed_header::SignedHeader, block::header::Header>
```

Comment:
can we use `lite::types::SignedHeader<C, H> where C: ..., H: ...` here and below?

Reply:
`where` expects traits, so we can say e.g. `where H: block::header::Header`. I can intro a type alias, or an import alias to make this shorter and more readable though.

Reply:
https://github.com/interchainio/tendermint-rs/pull/141/commits/be65629e64b3d7aecf50d94b39147ab9ba61d5ba